### DISCUSSION TITLE
part-ii/2013-88/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#110] [Question comments]
Part II 2013 Logic and Set Theory: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2013-88/

**Comment by ArthurConmy** [comment]
Bookwork, then Cantor Normal Form part is example sheet question, then the final answer is only the ordinals with Cantor Normal Form $\omega^{\beta_1}.b_1$. 

CLAIM 1: For $\alpha < \beta$ ordinals, $\omega^{\alpha} + \omega^{\beta} = \omega^{\beta}$.

CLAIM 2: The result follows from CLAIM 1.

PROOF 2: Write out $\omega^{\beta_1}.b_1 + ... + \omega^{\beta_n}.b_n$, then by CLAIM 1 and associativity, this is just $\omega^{\beta_n}.b_n$.

PROOF 1: Write $\omega^{\beta} = \omega^{\alpha} ( 1 + \omega^{ \beta - \alpha } ) = \omega^\alpha \omega^{ \beta-\alpha }$ since for $\beta - \alpha \ge 1$ means adding one at the start has no effect since we can biject that $1$ into the first $\omega$ copy.
